cg/services/illumina_services/illumina_metrics_service/illumina_demux_version_service.py

```diff
@@ -0,0 +1,35 @@
+"""Module to parse the illumina demultiplexing log file."""
+
+from pathlib import Path
+
+from pydantic import BaseModel, Field
+
+from cg.constants import SPACE
+from cg.constants.constants import FileFormat
+from cg.io.controller import ReadFile
+from cg.io.json import read_json
+
+
+class DemuxSoftware(BaseModel):
+    software_version: str = Field(..., alias="dragen_version")
+    software: str = Field(default="dragen")
```

Comment:
I think we don't really have any other software for illumina anymore, right? It is all dragen that uses BCLConvert?

Reply:
Yes

Reply:
But if we want to be consistent, the name of the software would be BCLConvert. There is the dragen version and the BCLConvert version